include/mbedtls/platform.h

```diff
@@ -311,33 +314,33 @@ mbedtls_platform_context;
 #endif /* !MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT */
 
 /**
- * \brief   This function performs any platform initialization operations.
+ * \brief   This function performs any platform-specific initialization operations.
```

Comment:
The description of this function is inadequate and confusing. I'd like to propose extending this description:

> This function performs platform-specific initialization operations, and on platforms that support this function, must be called before all other library functions.
> 
> The function is intended to allow platforms that support cryptographic hardware acceleration to initialise their hardware before first use.
> 
> The function is optional, and not all platforms will support it. The default implementation of the platform abstraction does not require this function to be called.

Reply:
Added your expansion as detailed file description. Thank you!